Formula/rstudio-server.rb

```diff
@@ -80,15 +84,11 @@ def which_linux_distribution
     distritbuion
   end
 
-  # fix build failure with latest macOS compiler
-  patch do
-    url "https://github.com/rstudio/rstudio/commit/b3b1ef10ca8473b6cf02b22cdb57f871f0237a02.diff?full_index=1"
-    sha256 "37c58b7f31307db369ba8c5269f5c303838e06d5db44429766b0213ea6eb6c34"
-  end
-
   def install
-    # Reduce memory usage below 4 GB for CI.
-    ENV["MAKEFLAGS"] = "-j2" if ENV["CIRCLECI"] || ENV["TRAVIS"]
+    if ENV["CIRCLECI"] || ENV["TRAVIS"]
```

Comment:
```suggestion
    if ENV["CI"]
```
It looks like this `CIRCLECI` environment variable is being removed by brew. See https://github.com/Homebrew/brew/blob/3944416c4602ae8618c4da399743fe96ddf0cf6a/bin/brew#L82